app/src/main/java/com/bunbeauty/fooddeliveryadmin/compose/element/SelectableItemView.kt

```diff
@@ -0,0 +1,87 @@
+package com.bunbeauty.fooddeliveryadmin.compose.element
+
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.bunbeauty.fooddeliveryadmin.R
+import com.bunbeauty.fooddeliveryadmin.compose.element.card.AdminCard
+import com.bunbeauty.fooddeliveryadmin.compose.icon16
+import com.bunbeauty.fooddeliveryadmin.compose.theme.AdminTheme
+
+@Composable
+fun SelectableItemView(
```

Comment:
Let's name `SelectableItem`

Reply:
It might make sense to create a special model for this Composable.
```
SelectableItem(
  uuid: String,
  title: String,
  isSelected: String,
)
```